runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -6583,9 +6663,28 @@ static void jitReleaseCodeStackWalk(OMR_VMThread *omrVMThread, condYieldFromGCFu
       do
          {
          thr->dropFlags &=0x0;
+#if JAVA_SPEC_VERSION >= 19
+         if (NULL != thr->currentContinuation) {
+            thr->currentContinuation->dropFlags &=0x0;
```

Comment:
Why `&= 0` instead of just `= 0` (lines 6665, 6668, 6683)?

Reply:
I think it was written like this originally as the inverse of `->dropFlags |= 0x1`;

Will update this to `= 0` as part of the upcoming refactor PR (to avoid duplicate walk for mounted continuation)

Reply:
If the intent was just to clear the low-order bit, then the code is wrong (it clears all bits), it should be
```c
            thr->currentContinuation->dropFlags &= ~1;
```